javascript/test/wombatUrlRewriting.js

```diff
@@ -0,0 +1,852 @@
+import test from 'ava';
+
+import { urlRewriteFunction } from '../src/wombatSetup.js';
+import { createGzip } from 'zlib';
+
+test.beforeEach((t) => {
+  t.context.prefix = 'http://library.kiwix.org/content/myzim_yyyy-mm/';
+  t.context.originalHost = 'www.example.com';
+  t.context.originalScheme = 'https';
+  t.context.documentPath = '/path1/resource1.html';
+  t.context.originalUrl =
+    t.context.originalScheme +
+    '://' +
+    t.context.originalHost +
+    t.context.documentPath;
+  t.context.currentUrl =
+    t.context.prefix + t.context.originalHost + t.context.documentPath;
+});
+
+test('undefined', (t) => {
+  t.is(
+    urlRewriteFunction(
+      t.context.currentUrl,
+      t.context.originalHost,
+      t.context.originalScheme,
+      t.context.originalUrl,
+      t.context.prefix,
+      undefined,
+      undefined,
+      undefined,
+      undefined,
+    ),
+    undefined,
+  );
+});
+
+// Why? I don't get it ... this is not supposed to be a valid URL ...
```

Comment:
This is a relative url (probably not intended by user, but still handle as is). So nothing to transform ?

Reply:
It should be transformed as well, this is why I don't get why it was not transformed previously